src/hotspot/share/classfile/systemDictionary.cpp

```diff
@@ -1125,31 +1123,20 @@ InstanceKlass* SystemDictionary::resolve_from_stream(Symbol* class_name,
   // If a class loader supports parallel classloading, handle parallel define requests.
   // find_or_define_instance_class may return a different InstanceKlass
```

Comment:
How about adding "// find_or_define_instance_class may return a different InstanceKlass, in which case the old k would be deallocated"

Reply:
Ok.